### PR TITLE
Update READMEs for migratestatus and metrics-bigquery.

### PR DESCRIPTION
**Pull request by cjwagner**
/cc @fejta

### CODE REVIEW

File: maintenance/migratestatus/README.md [hunk 1]
```diff
@@ -25,6 +25,6 @@ The migratestatus binary is run locally and can be built by running `go build` f
 - `--dest` The destination context. This is the context that is copied **to** and/or the context that replaces the retired context. This flag may only be omitted if retire mode is specified and the old context is being retired without a replacement.
 
 The binary also accepts the following optional parameters:
```
Comment (fejta) [inline maintenance/migratestatus/README.md]: Wouldn't it just be easier (and more forwards compatible) to tell people to `go run k8s.io/maintenance/migratestatus -h` to read all this?

File: metrics/README.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
 # Bigquery metrics
 
-This folder contains config files describing metrics that summarize data
-in our Bigquery test result database. Config files are consumed
+This 'metrics-bigquery' job generates metrics that summarize data in our Bigquery
```
Comment (fejta) [inline metrics/README.md]: Consider using a backtick instead of tick

File: metrics/README.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 by the `metrics-bigquery` periodic prow job that runs the bigquery
-image and scenario.  Each metric consists of a bigquery query and
-a jq filter. The query is run every 24 hours to produce a json data
+image and python script.  Each metric config is a yaml file consisting of a bigquery query, a jq filter
+to filter the results, and possibly another jq filter to convert the query
+results to a JSON representation of influxdb timeseries data points.
+
+Each query is run every 24 hours to produce a json
 file containing the complete raw query results named with the format
 'raw-yyyy-mm-dd.json'. The raw file is then filtered with the associated
```
Comment (fejta) [inline metrics/README.md]: Backtick

File: metrics/README.md [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
 jq filter and the results are stored in 'daily-yyyy-mm-dd.json'.  These
```
Comment (fejta) [inline metrics/README.md]: backtick

File: metrics/README.md [hunk 5]
```diff
@@ -13,6 +16,10 @@ the metric name and persist for a year after their creation. Additionally,
 the latest filtered results for a metric are stored in the root of the
 k8s-metrics bucket and named with the format 'METRICNAME-latest.json'.
```
Comment (fejta) [inline metrics/README.md]: backtick

File: metrics/README.md [hunk 6]
```diff
@@ -1,10 +1,13 @@
 # Bigquery metrics
```
Comment (fejta) [inline metrics/README.md]: Personally rather reading this Tolkienesque snippet (that I'm unlikely to read anyway unless as a last resort) I would prefer a couple concrete examples and an explanation for each piece

**Comment by cjwagner** [comment]
@fejta I added an example and moved the details about when and where results are stored out of the introduction. Also fixed the backticks.

**Pull request merged**